<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_devais</class>
 <widget class="QWidget" name="add_devais">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>42</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>40</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);
color:rgb(238, 238, 236);</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="electr">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>108</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);
color:rgb(238, 238, 236);</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="gas">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>165</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);
color:rgb(238, 238, 236);</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="water">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>230</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);
color:rgb(238, 238, 236);</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="warning_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>281</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="warning_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>281</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="warning_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>281</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="warning">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>281</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="enter">
   <property name="geometry">
    <rect>
     <x>107</x>
     <y>345</y>
     <width>173</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>107</x>
     <y>385</y>
     <width>173</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>386</width>
     <height>436</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pich.qrc">:/picha/Pichures/devais.png</pixmap>
   </property>
  </widget>
  <zorder>label_5</zorder>
  <zorder>label</zorder>
  <zorder>name</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>electr</zorder>
  <zorder>gas</zorder>
  <zorder>water</zorder>
  <zorder>checkBox</zorder>
  <zorder>warning_2</zorder>
  <zorder>warning_3</zorder>
  <zorder>warning_4</zorder>
  <zorder>warning</zorder>
  <zorder>enter</zorder>
  <zorder>back</zorder>
 </widget>
 <resources>
  <include location="pich.qrc"/>
 </resources>
 <connections/>
</ui>
